### PR TITLE
Convert all addressbook references into Flash Notes

### LINKED ISSUE #56: Rename AddressBook

### PR DESCRIPTION
**Pull request by chuyouchia**

### CODE REVIEW

File: src/main/java/seedu/flashnotes/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
     @Override
```
Comment (foongsq) [inline src/main/java/seedu/flashnotes/logic/commands/ClearCommand.java:17]: Perhaps the "Address book" in MESSAGE_SUCCESS should be changed to "Flash notes"?

File: src/main/java/seedu/flashnotes/logic/commands/ExitCommand.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
-package seedu.address.logic.commands;
+package seedu.flashnotes.logic.commands;
 
-import seedu.address.model.Model;
+import seedu.flashnotes.model.Model;
 
 /**
  * Terminates the program.
```
Comment (foongsq) [inline src/main/java/seedu/flashnotes/logic/commands/ExitCommand.java:6]: Perhaps the "Address Book" in MESSAGE_EXIT_ACKNOWLEDGEMENT could be changed to "Flash Notes"?

**Review by foongsq — CHANGES_REQUESTED** [review]
Almost ready to merge, there are 2 instances of "address book" that still needs to be converted to "flash notes".

File: src/main/java/seedu/flashnotes/logic/commands/ClearCommand.java (hunk 1 above)
Comment (chuyouchia) [reply src/main/java/seedu/flashnotes/logic/commands/ClearCommand.java:17]: Done, thanks for the catch!

**Review by foongsq — APPROVED** [review]
LGTM

**Pull request merged**